source/_patterns/01-molecules/15-loan-sections/18-loan-story.mustache

```diff
@@ -67,7 +67,7 @@
 					<img class="reviewer-image" src="{{reviewerImageLink}}" alt="Translator profile picture"/>
 				{{/reviewerImageLink}}
 				<p>
-					Translated from {{language}} by {{{reviewerLink}}} {{reviewerName}}
+					Translated from {{language}} {{#reviewerLink}} by {{{reviewerLink}}} {{reviewerName}}{{/reviewerLink}}
```

Comment:
@jedlin-kiva Ahh right, indeed, the original case has differed now a bit from where it's at this point, but you're correct it should look at name and show both if name is available, otherwise show neither. Fix again coming right up.